### DISCUSSION TITLE
Feature Request: Some form of account linking

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [bookwyrm-social/bookwyrm#1324] [Ideas]
It would be fantastic if there was some way to link with other Fediverse accounts. This way one can have a consistent identity across multiple services. Any of the following would be significant progress towards that goal:

1. The ability to log in using an account on another Fediverse service, using OAUTH, OpenID, ActivityPub, or whatever is feasible
2. The ability to force a boost on remote ActivityPub accounts, such that when you post here, it is reflected there

**Comment by mouse-reeve** [comment]
Thank you! I think #1236 is related

**Reply by roke-julian-lockhart** [reply]
@mouse-reeve, https://github.com/bookwyrm-social/bookwyrm/discussions/1324#discussioncomment-5798044

**Reply by dato** [reply]
> (https://github.com/bookwyrm-social/bookwyrm/issues/1236 doesnt exist)

Possibly #2379, then?

**Comment by donpdonp** [comment]
is there an update on this? (#1236 doesnt exist). Its the first thing I thought of when coming to the site - create another account? I'd like to login with my mastodon account.
